airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -942,6 +968,15 @@ def __init__(self,
         self.schema_object = schema_object
 
         # BQ config
+        if not table_resource:
```

Comment:
Should we raise exception when user provide mutully exclussive parameters e.g. table_resource and source_uris?

Reply:
I agree. the behaviour is a bit undefined if wrong combination of  parameters is specified. If table_resources is undefined and schema_fields are defined and source_objects and bucket  are, I am not sure what happens here. I think all the "superfluous" combinations where same data is provided by different means  should result in exception

Reply:
I've added an exception